app/cmd/backup.go

```diff
@@ -286,6 +295,15 @@ func createBackup(c *cli.Context) error {
 	url := c.GlobalString("url")
 	volumeName := c.GlobalString("volume-name")
 	engineInstanceName := c.GlobalString("engine-instance-name")
+
+	backupBlockSizeMB := c.Int64("backup-block-size")
+	if backupBlockSizeMB <= 0 {
+		return errors.New(fmt.Sprintf("invalid backup block size: %v", backupBlockSizeMB))
+	} else {
+		quantity := resource.NewQuantity(backupBlockSizeMB*1024*1024, resource.BinarySI)
+		parameters[lhbackup.LonghornBackupParameterBackupBlockSize] = strconv.FormatInt(quantity.Value(), 10)
```

Comment:
Just curious. Do we need the quantity here?

Reply:
Personally prefer the quantity because of readability. The parameters will be saved into the backup's metadata JSON file, and could be checked without any tool.